packages/chart/src/components/EditorPanel.js

```diff
@@ -554,7 +554,7 @@ const EditorPanel = () => {
       {undefined === config.newViz && config.runtime && config.runtime.editorErrorMessage && <Error /> }
       <button className={displayPanel ? `editor-toggle` : `editor-toggle collapsed`} title={displayPanel ? `Collapse Editor` : `Expand Editor`} onClick={onBackClick}></button>
       <section className={`${displayPanel ? 'editor-panel' : 'hidden editor-panel'}${isDashboard ? ' dashboard': ''}`}>
-        <div className="heading-2">Configure Chart</div>
+        <div role="heading" className="heading-2">Configure Chart</div>
```

Comment:
Hey @ShawnStringfield ,

Can you add `aria-level="2"` here? Thanks!

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role#description